src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/common/matmul_decompress_convert.cpp

```diff
@@ -519,5 +514,141 @@ INSTANTIATE_TEST_SUITE_P(smoke_FC_2D_FP16_2,
 
 }  // namespace
 
+
+/* This test covers NNCF-case when decompression convert has not only MatMul consumer.
+ * Graph before:
+   ------------             ---------------
+   |Input(f32)|             |Constant(f16)|
+   ------------             ---------------
+        |                         |
+        |         ---------------------------------
+        |         |Convert(decompression f16->f32)|
+        |         ---------------------------------
+        |             |                       | 
+    ----------------------------        -----------------------
+    |MatMul (transposed_b=true)|        |       Result        |
+    ----------------------------        -----------------------
+              |
+    -----------------------
+    |       Result        |
+    -----------------------
+
+ * Exec graph:
+   ------------     -----------------------------
+   |Input(f32)|     |       Constant(f32)       |
+   ------------     -----------------------------
+        |             |                   |
+        |        -------------      -----------------------
+        |        | Transpose |      |       Result        |
+        |        -------------      -----------------------
+        |             |
+    -----------------------
+    |    FullyConnected   |
+    -----------------------
+              |
+    -----------------------
+    |       Result        |
+    -----------------------
+*/
```

Comment:
Theoretically, we can have the following exec graph:
```
   ------------     -----------------------------
   |Input(f32)|     |       Constant(f16)       |
   ------------     -----------------------------
        |             |                   |
        |        -------------      ---------------------------------
        |        | Transpose |      | Convert(f16->f32) + Result   |
        |        -------------      ---------------------------------
        |             |
    -----------------------
    |    FullyConnected   |
    -----------------------
              |
    -----------------------
    |       Result        |
    -----------------------
```
Wouldn't we benefit from this? If yes, let's discuss possible options offline

Reply:
Yes, I agree that we can handle this case by your way. You're totally right.
But we discussed offline with @dmitry-gorokhov and decided that the implementation of this way (with `Convert`  copying) is a feature for the next generations 😄 At the moment, provided solution is enough for us.

Thanks!

Reply:
Discussed offline: @v-Golubev thank you for the idea! Implemented your subgraph behavior. Tests are green, everything works. Looks like I mixed up another discussion about `Convert` copying - yesterdays we discussed this copy to support `FullyConnectedCompressed` - it will be implemented in future if needed.

Reply:
Perfect, thank you!